### PR TITLE
Adding CTA pattern and my name to contributor list

### LINKED ISSUE #114: Block Patterns - Centered Call to Action
**Description**

🛠️ The goal of this issue is to track the implementation of the "Centered Call to Action" pattern.
🎨 [Figma](https://www.figma.com/file/AlYr03vh4dVimwYwQkTdf6/Twenty-Twenty-Four?type=design&node-id=492%3A4337&mode=design&t=LHSPHKUrPq556l59-1)

Centered Call to Action

### PR DESCRIPTION
**Pull request by LynUrth**

### CODE REVIEW

**Comment by chrisdesrochers** [comment]
Great job! 👍

File: patterns/cta.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Slug: twentytwentyfour/cta
+ * Categories: text
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"","padding":{"top":"100px","right":"100px","bottom":"100px","left":"100px"}}},"backgroundColor":"base","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide has-base-background-color has-background" style="padding-top:100px;padding-right:100px;padding-bottom:100px;padding-left:100px"><!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"50px","bottom":"50px"}}},"backgroundColor":"tertiary","layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide has-tertiary-background-color has-background" style="padding-top:50px;padding-bottom:50px"><!-- wp:paragraph {"align":"center","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"typography":{"fontSize":"28px","lineHeight":"1.6"}},"textColor":"contrast","fontFamily":"cardo"} -->
+<p class="has-text-align-center has-contrast-color has-text-color has-link-color has-cardo-font-family" style="font-size:28px;line-height:1.6">Join 900+ designers</p>
```
Comment (luminuu) [inline patterns/cta.php:13]: Strings need to be made translateable with the `esc_html__` or similar functions. You can look at other patterns how to do that.

File: patterns/cta.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"align":"center","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"typography":{"fontSize":"16px"}},"textColor":"contrast","fontFamily":"system-font"} -->
+<p class="has-text-align-center has-contrast-color has-text-color has-link-color has-system-font-font-family" style="font-size:16px">Stay in the loop with everything you need to know</p>
```
Comment (luminuu) [inline patterns/cta.php:17]: See above

File: patterns/cta.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons {"align":"wide","layout":{"type":"flex","justifyContent":"center"}} -->
+<div class="wp-block-buttons alignwide"><!-- wp:button {"width":25} -->
+<div class="wp-block-button has-custom-width wp-block-button__width-25"><a class="wp-block-button__link wp-element-button">Sign up</a></div>
```
Comment (luminuu) [inline patterns/cta.php:22]: See above

**Review by luminuu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**